docs/docs/how-tos/index.md

```diff
@@ -91,6 +91,12 @@ These how-to guides show common patterns for tool calling with LangGraph:
 - [How to view and update state in subgraphs](subgraphs-manage-state.ipynb)
 - [How to transform inputs and outputs of a subgraph](subgraph-transform-state.ipynb)
 
+### Multi-agent
+
+- [How to build a multi-agent network](multi-agent-network.ipynb)
+
+For more examples of multi-agent architectures implementations check out [these tutorials](../tutorials/index.md#multi-agent-systems).
```

Comment:
Could we avoid using `this guide` / `these tutorials` / `here` for the names of links?

See the [multi-agent tutorials](...) for implementations of other multi-agent architectures.